prototype2/common/ReadoutSerializer.cpp

```diff
@@ -42,20 +48,18 @@ int ReadoutSerializer::produce() {
   entries = 0;
   builder.Clear();
 
-  return buffersize;
+  return buffer.size;
```

Comment:
size is more like the  number of entries?

Reply:
So, I guess this depends on what we want it to return? Entries? Bytes? The specification is unclear, e.g. no explanation of what the return value should be in the h file.